behave_django/environment.py

```diff
@@ -21,9 +22,15 @@ def before_scenario(self, context):
         context.test.setUpClass()
         context.test()
 
-        context.base_url = context.test.live_server_url
+        context.base_url = getattr(context.test, 'live_server_url', None)
```

Comment:
I was considering to make `base_url` a `@property` but it would at least require monkey patching, so I thought it's too much of black magic. But let me try if it works at all...

Btw, any idea of what exception suits better than `ImproperlyConfigured`?